protobuf/syft_proto/execution/v1/plan.proto

```diff
@@ -3,15 +3,15 @@ syntax = "proto3";
 package syft_proto.execution.v1;
 option java_package = "org.openmined.syftproto.execution.v1";
 
-import "syft_proto/execution/v1/operation.proto";
+import "syft_proto/execution/v1/computation_action.proto";
 import "syft_proto/execution/v1/state.proto";
 import "syft_proto/frameworks/torch/tensors/interpreters/v1/placeholder.proto";
 import "syft_proto/types/syft/v1/id.proto";
 
 
 message Plan {
     syft_proto.types.syft.v1.Id id = 1;
-    repeated syft_proto.execution.v1.Operation operations = 2;
+    repeated syft_proto.execution.v1.ComputationAction operations = 2;
```

Comment:
Maybe we should rename this field to `actions`?

Reply:
Totally!